### PR TITLE
feat(Title): Implement Title component

### PR DESCRIPTION
**Pull request by renatofreire**
# What

Add Title component

# Why

Title component was the last missing typography

### CODE REVIEW

**Review by pedro-cerdera — DISMISSED** [review]

**Review by victorheringer — APPROVED** [review]
LGTM

File: src/components/Text/Title.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+
+import { poppinsBold } from '@tokens/fonts';
+import { BaseTextProps } from './types';
```
Comment (LopesAlysson) [inline src/components/Text/Title.tsx:5]:
```suggestion
import type { BaseTextProps } from './types';
```

File: src/components/Text/Title.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import BaseText from './BaseText';
+
+function BaseTitle({
+  color = colors.moon900,
+  align = 'center',
+  fontSize,
+  children,
+  ...props
+}: BaseTextProps & { fontSize: number; bold: boolean }) {
```
Comment (LopesAlysson) [inline src/components/Text/Title.tsx:14]: Is it possible to have a non-bold Title?

**Review by LopesAlysson — CHANGES_REQUESTED** [review]

File: src/components/Text/Title.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  color = colors.moon900,
+  align = 'center',
+  fontSize,
+  children,
+  ...props
+}: BaseTextProps & { fontSize: number }) {
+  const baseProps = {
+    color,
+    align,
+    fontFamily: poppinsSemiBold,
```
Comment (renatofreire) [inline src/components/Text/Title.tsx:18]: @LopesAlysson good point. Actually, all titles are semibold.
I fixed it and remove the unnecessary prop :)

**Review by LopesAlysson — APPROVED** [review]
:shipit:

**Pull request merged**